_posts/payalpatel/2016-04-06-PayalsMOARDictionaryEx.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+author: payalpn
+title: "Payal's MOAR Dictionary Exercises"
+---
+
+**Exercise 1:**
+
+<iframe src="https://trinket.io/embed/python3/e8d7f60b08" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Exercises 2-5:**
+
+<iframe src="https://trinket.io/embed/python3/a2178ed680" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
I don't see your code in here, just the original.  Did you get this embed code via the share menu?